hw/top_darjeeling/data/autogen/defs.bzl

```diff
@@ -103,35 +103,6 @@ DARJEELING_ALERTS = [
     "clkmgr_aon_fatal_fault",
     "pinmux_aon_fatal_fault",
     "aon_timer_aon_fatal_fault",
-    "soc_proxy_fatal_alert_intg",
-    "soc_proxy_fatal_alert_external_0",
-    "soc_proxy_fatal_alert_external_1",
-    "soc_proxy_fatal_alert_external_2",
-    "soc_proxy_fatal_alert_external_3",
-    "soc_proxy_fatal_alert_external_4",
-    "soc_proxy_fatal_alert_external_5",
-    "soc_proxy_fatal_alert_external_6",
-    "soc_proxy_fatal_alert_external_7",
-    "soc_proxy_fatal_alert_external_8",
-    "soc_proxy_fatal_alert_external_9",
-    "soc_proxy_fatal_alert_external_10",
-    "soc_proxy_fatal_alert_external_11",
-    "soc_proxy_fatal_alert_external_12",
-    "soc_proxy_fatal_alert_external_13",
-    "soc_proxy_fatal_alert_external_14",
-    "soc_proxy_fatal_alert_external_15",
-    "soc_proxy_fatal_alert_external_16",
-    "soc_proxy_fatal_alert_external_17",
-    "soc_proxy_fatal_alert_external_18",
-    "soc_proxy_fatal_alert_external_19",
-    "soc_proxy_fatal_alert_external_20",
-    "soc_proxy_fatal_alert_external_21",
-    "soc_proxy_fatal_alert_external_22",
-    "soc_proxy_fatal_alert_external_23",
-    "soc_proxy_recov_alert_external_0",
-    "soc_proxy_recov_alert_external_1",
-    "soc_proxy_recov_alert_external_2",
-    "soc_proxy_recov_alert_external_3",
```

Comment:
Shouldn't this removal be part of the previous commit which deals with the alerts?